email_automation/email-config.py

```diff
@@ -4,40 +4,57 @@
 import os
 import logging
 import sys
+import argparse
 logger = logging.getLogger()
 logger.setLevel(logging.INFO)
 # If you don't have boto3 installed, make sure to `pip install boto3` before running this script. 
 
-#Input the filename as an argument in command line 
 if __name__ == "__main__":
-    filename_raw = sys.argv[1]
-    filename = filename_raw.split(".")[0]
+    parser = argparse.ArgumentParser()
+    group = parser.add_mutually_exclusive_group(required=True)
+    group.add_argument('-l', '--local',
+                       help = 'Running locally. Provide full path to config file + install boto3 prior to running.' )
+    group.add_argument('-g', '--github',
+                       help = 'Must be run on GitHub. To run locally, use -l argument.') 
+    args = parser.parse_args()
+    filepath_raw = sys.argv[2]
+    filename_raw = filepath_raw.split("/")[-1]
+    filename = filename_raw.split('.')[0]
     pool_name = "nrelopenpath-prod-" + filename
-    # pool_name = 'nrelopenpath-stage'
     current_path = os.path.dirname(__file__)
-    config_path = os.path.relpath('../configs/'+ filename_raw, current_path)
+    maindir = current_path.rsplit("/",1)[0]
+    config_path = os.path.relpath('../configs/'+ filename_raw, current_path) if args.local else maindir + f'/configs/{filename_raw}'
+    print("config_path", config_path)
 
-#Set up AWS credentials as environment variables + set variables 
-ACCESS = os.environ.get("AWS_ACCESS_KEY_ID")
-SECRET = os.environ.get("AWS_SECRET_ACCESS_KEY")
-TOKEN = os.environ.get("AWS_SESSION_TOKEN")
-AWS_REGION = "us-west-2"
+if args.local:
+    #Set up AWS credentials as environment variables + set variables 
+    ACCESS = os.environ.get("AWS_ACCESS_KEY_ID")
+    SECRET = os.environ.get("AWS_SECRET_ACCESS_KEY")
+    TOKEN = os.environ.get("AWS_SESSION_TOKEN")
+    AWS_REGION = "us-west-2"
+    welcome = 'welcome-template.txt'
```

Comment:
where is this file? I don't see a `welcome.txt` in the PR here and this seems to imply that it is in the same directory as the script...

Reply:
See [comment](https://github.com/e-mission/nrel-openpath-deploy-configs/pull/45#discussion_r1433335001)

Reply:
The variable `welcome` changes between -l and -g, if that's what you're referencing!